Doc/library/asyncio-task.rst

```diff
@@ -527,6 +527,41 @@ Running Tasks Concurrently
       and there is no running event loop.
 
 
+Eager Task Factory
+==================
+
+.. function:: eager_task_factory(loop, coro, *, name=None, context=None)
+
+    A task factory for eager task execution.
+
+    When using this factory (via ``loop.set_task_factory(asyncio.eager_task_factory)``),
+    coroutines that are able to complete synchronously (without blocking)
+    are returned immediately as a finished :class:`Task`.
+
+    This task factory attempts to execute the coroutine ``coro`` immediately
```

Comment:
```suggestion
    The eager task factory attempts to execute the coroutine ``coro`` immediately before
    scheduling the task to the event loop. The coroutine will execute until it blocks, returns,
    or raises. If the coroutine blocks, the pending :class:`Task` is scheduled to the event loop
    and returned. If coroutine returns or raises, a finished :class:`Task` is returned, and the task
    is not scheduled to the event loop.
    
```

Reply:
@willingc does the latest version of this paragraph look good to you?

Reply:
Yup. Looks good @itamaro.